src/ext/java/org/opentripplanner/ext/flex/trip/FlexTrip.java

```diff
@@ -40,7 +39,13 @@ public FlexTrip(Trip trip) {
 
   public abstract int latestArrivalTime(int arrivalTime, int fromStopIndex, int toStopIndex, int flexTime);
 
-  public abstract Collection<StopLocation> getStops();
+  /**
+   * Returns all the stops that are in this trip.
+   *
+   * Note that they are in no specific order and don't correspond 1-to-1 to the stop times of the
+   * trip.
```

Comment:
Maybe something explicit about the fact that groups are exploded to their individual members.

Reply:
Good point. I changed the comment.